.github/workflows/deploy-vercel-staging.yml

```diff
@@ -269,19 +245,27 @@ jobs:
       - uses: actions/checkout@v1 # Get last commit pushed - See https://github.com/actions/checkout
       - name: Expose GitHub slug/short variables # See https://github.com/rlespinasse/github-slug-action#exposed-github-environment-variables
         uses: rlespinasse/github-slug-action@v3.x # See https://github.com/rlespinasse/github-slug-action
+
+      # Restore variables stored in previous jobs
+      - name: Restore variables
+        uses: UnlyEd/github-action-store-variable@v1.0.1 # See https://github.com/UnlyEd/github-action-store-variable
+        id: restore-variable
+        with:
+          variables: |
+            CUSTOMER_REF_TO_DEPLOY
+
       - name: Expose git environment variables and call webhook (if provided)
         # Workflow overview:
         #  - Resolves webhook url from customer config file
         #  - If a webhook url was defined in the customer config file, send an HTTP request, as POST request, with a JSON request body dynamically generated
         #  - Prints the headers of the POST HTTP request (curl)
+        # TODO Convert this into an external bash script if possible, to simplify this step as much as possible
         run: |
-          MANUAL_TRIGGER_CUSTOMER="${{ github.event.inputs.customer}}"
-          CUSTOMER_REF_TO_DEPLOY="${MANUAL_TRIGGER_CUSTOMER:-$(cat vercel.json | jq --raw-output '.build.env.NEXT_PUBLIC_CUSTOMER_REF')}"
-
           VERCEL_DEPLOYMENT_COMPLETED_WEBHOOK=$(cat vercel.$CUSTOMER_REF_TO_DEPLOY.staging.json | jq --raw-output '.build.env.VERCEL_DEPLOYMENT_COMPLETED_WEBHOOK')
+          echo "Vercel deployment webhook url: " $VERCEL_DEPLOYMENT_COMPLETED_WEBHOOK
 
           # Checking if a webhook url is defined
-          if [ -n "$VERCEL_DEPLOYMENT_COMPLETED_WEBHOOK" ]; then
+          if [ -n  "$VERCEL_DEPLOYMENT_COMPLETED_WEBHOOK" ]; then
```

Comment:
This might be outside this PR's scope, but if you know of any way to extract it as an external bash script, and call it as a package.json script, that'd be a great improvement for readability.